README.rst

```diff
@@ -61,3 +61,10 @@ from KaTeX for auto-rendered content are:
         {left: "\\[", right: "\\]", display: true}
     ]
     '''
+
+The version of KaTeX used is controlled by the ``katex_version`` config setting,
+and the specific delimiters written to the HTML when math mode is encountered
+are controlled by the ``katex_inline`` and ``katex_display`` config settings.
+By default, ``katex_inline`` is ``[r'\(', r'\)']`` and ``katex_display`` is
+``[r'\[', r'\]']``. If you change these, make sure to update the ``delimiters``
+by adding the ``katex_options`` config setting.
```

Comment:
After deciding on the handling of the options we should mention the general option handling in the README provide a link to its deswcription in the KaTeX README and then provide the LaTeX macros and the delimiters as two examples of the option handling.

Reply:
Same as below for the options, I'll let you handle this as you see fit.